CPT331.Core/ObjectModel/Coordinate.cs

```diff
@@ -79,16 +98,30 @@ public static bool IsLongitudeValid(double longitude)
 			);
 		}
 
+        [Obsolete("Use IsValid(double, double) instead.")]
 		public static bool IsValid(Coordinate value)
 		{
 			return IsValid(value._latitude, value._longitude);
 		}
 
+        /// <summary>
+        /// Determines whether the coordindate values specified are valid.
+        /// </summary>
+        /// <param name="latitude">The angular distance of a place north or south of the earth's equator, ranging from -90 to 90.</param>
+        /// <param name="longitude">The angular distance of a place east or west of the Greenwich meridian, ranging from -180 to 180.</param>
+        /// <returns>Returns true if both values are valid; otherwise false.</returns>
 		public static bool IsValid(double latitude, double longitude)
 		{
 			return (IsLatitudeValid(latitude) && IsLongitudeValid(longitude));
 		}
 
+        /// <summary>
+        /// Factory method that constructs Coorindate object using the specified values.
+        /// </summary>
+        /// <param name="latitude">The angular distance of a place north or south of the earth's equator, ranging from -90 to 90.</param>
+        /// <param name="longitude">The angular distance of a place east or west of the Greenwich meridian, ranging from -180 to 180.</param>
+        /// <exception cref="ArgumentException">Values specified do not translate into valid geographical coordinates.</exception>
+        /// <returns>An instance of Coorindate if the specified values are valid; otherwise null.</returns>
```

Comment:
Summary documentation. Again on the fence whether or not to continue.

It may add value in some situations. In this case, information about the potential for Exception is useful.